task8.css

```diff
@@ -0,0 +1,42 @@
+body {
+    background-color: #000000;
```

Comment:
let's use 2 spaces on all documents always 
![image](https://user-images.githubusercontent.com/40792121/77566676-70f3a700-6edf-11ea-98dd-3a1b32707723.png)

Reply:
you can update settings in your vs code

Reply:
please fix spaces